molgenis-core-ui/src/main/java/org/molgenis/ui/MolgenisWebAppConfig.java

```diff
@@ -101,6 +101,10 @@ public void addResourceHandlers(ResourceHandlerRegistry registry)
 				.setCachePeriod(cachePeriod);
 		registry.addResourceHandler("/generated-doc/**").addResourceLocations("/generated-doc/").setCachePeriod(3600);
 		registry.addResourceHandler("/html/**").addResourceLocations("/html/", "classpath:/html/").setCachePeriod(3600);
+
+		// Swag
+		registry.addResourceHandler("/swagger-ui/**")
+				.addResourceLocations("classpath:/META-INF/resources/webjars/swagger-ui/2.2.8/");
```

Comment:
Can we change this such that changing the maven dependency swagger-ui version will not break the swagger UI?